backends/cxxrtl/cxxrtl_backend.cc

```diff
@@ -3401,6 +3405,10 @@ struct CxxrtlBackend : public Backend {
 				worker.design_ns = args[++argidx];
 				continue;
 			}
+			if (args[argidx] == "-print-output" && argidx+1 < args.size()) {
+				worker.print_output = args[++argidx];
```

Comment:
Can you add a restriction here so that it can only be `std::cout` or `std::cerr`? This way we won't have to add backwards compat with all the ways people can (mis)use this option.

Reply:
Done.